roles/nginx/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+nginx_reverse_proxy_name: reverse_proxy
+nginx_site_port: 80
+nginx_server_name: ''
+nginx_dest_ip: 127.0.0.1
+nginx_dest_port: 8080
```

Comment:
Since this repo is specifically for installing and configuring a ZNC host, I think you may be right. I'll switch it to 5000.